content/bitcoin-core.md

```diff
@@ -2,26 +2,28 @@
 layout: default
 title: In Bitcoin Core
 permalink: /bitcoin-core/
-nav_order: 4
+nav_order: 3
 ---
 
 ## ASmap in Bitcoin Core
 
-Using an ASmap file in one's Bitcoin Core configuration is already strictly better than the default, and so we want to encourage usage of ASMap among users. However, currently the adoption of the ASMap feature must be assumed to be minimal, since doing so requires creating or procuring an ASMap file and then passing it as an option to the node during startup. Aside from the common wisdom that most users don't change the defaults, there is no trusted way to provide ASMap files for download.
+---------------------
+
+Using an ASmap file in one's Bitcoin Core configuration is already strictly better than the default, and so we want to encourage usage of ASmap among users. However, currently the adoption of the ASmap feature must be assumed to be minimal, since doing so requires creating or procuring an ASMap file and then passing it as an option to the node during startup. Aside from the common wisdom that most users don't change the defaults, there is no trusted way to provide ASmap files for download.
 
 ## Usage
 
-#### Passing an ASmap to Core
+#### Create an ASmap with Kartograf
 
-Core will accept a compressed ASmap file with the `-asmap` option as described [here](https://github.com/bitcoin/bitcoin/blob/master/doc/release-notes/release-notes-0.20.0.md#new-settings).
+[Kartograf](https://github.com/fjahr/kartograf) is a tool to fetch AS data from regional IRRs, and produce a file to be used with Core. Once we have a reliable process for publishing ASmaps, we'll make them available for public download (tbd).
 
-#### Creating an ASmap with Kartograf
+#### Compress it with `asmap-tool` in Core
 
-[Kartograf](https://github.com/fjahr/kartograf) is a tool to fetch AS data from regional IRRs, and produce a file to be used with Core.
+[asmap-tool](https://github.com/bitcoin/bitcoin/tree/master/contrib/asmap) is a Python script to help encode/compress an ASmap file before passing it to Core.
 
-#### Using `asmap-tool` in Core
+#### Pass the ASmap to Core
 
-[asmap-tool](https://github.com/bitcoin/bitcoin/tree/master/contrib/asmap) is a Python script to help encode/compress an ASmap file before passing it to Core.
+Core will accept a compressed ASmap file with the `-asmap` option as described [here](https://github.com/bitcoin/bitcoin/blob/master/doc/release-notes/release-notes-0.20.0.md#new-settings).
```

Comment:
Hm, I think it's better to point people to the bitcoins help than to older release notes that might be outdated. `src/bitcoind --help`

Reply:
Also, I think the first two steps could be marked as optional somehow. It's only needed if people want to create a file themselves. But if not they can just go to asmap-data and get the latest file there. We might even want to recommend that for inexperienced users because I am not 100% confident yet that kartograf really is "fool-proof" enough when used by an inexperience user. Say they experience some network disruption during the download phase and miss some files and kartograf doesn't error properly and instead produces an incomplete file. Today I can't guarantee that is impossible and looking at the statistics we print out may not make it obvious enough that something is out of the ordinary.

Reply:
yea i've clarified this [799cec5](https://github.com/fjahr/asmap-website/pull/9/commits/799cec5e29ae18391a900abbb04ef990c15ff0e5)